### PR TITLE
Follow-up to colocation PR

### PR DESCRIPTION
**Pull request by kgaillot**
This takes care of a few suggestions from #2773 and continues removing the "lh/rh" terminology from other types of constraints

### CODE REVIEW

**Comment by kgaillot** [comment]
@clumens , can you review this one when you get a chance? Low priority

File: lib/pacemaker/pcmk_sched_location.c [hunk 1]
```diff
@@ -197,19 +197,19 @@ generate_location_rule(pe_resource_t *rsc, xmlNode *rule_xml,
 }
 
 static void
-unpack_rsc_location(xmlNode *xml_obj, pe_resource_t *rsc_lh, const char *role,
+unpack_rsc_location(xmlNode *xml_obj, pe_resource_t *rsc, const char *role,
                     const char *score, pe_working_set_t *data_set,
```
Comment (clumens) [inline lib/pacemaker/pcmk_sched_location.c:201]: Do you have a future patch lined up to remove the data_set argument here?

File: lib/pacemaker/pcmk_sched_tickets.c [hunk 2]
```diff
@@ -43,117 +43,117 @@ typedef struct {
...
  *
- * \param[in] rsc_lh      Resource affected by ticket
+ * \param[in] rsc         Resource affected by ticket
  * \param[in] rsc_ticket  Ticket
  * \param[in] data_set    Cluster working set
  */
 static void
-constraints_for_ticket(pe_resource_t *rsc_lh, rsc_ticket_t *rsc_ticket,
+constraints_for_ticket(pe_resource_t *rsc, rsc_ticket_t *rsc_ticket,
                        pe_working_set_t *data_set)
```
Comment (clumens) [inline lib/pacemaker/pcmk_sched_tickets.c:66]: Likewise here.

File: lib/pacemaker/libpacemaker_private.h [hunk 3]
```diff
@@ -334,9 +334,9 @@ pcmk__colocation_has_influence(const pcmk__colocation_t *colocation,
 // Ordering constraints (pcmk_sched_ordering.c)
 
 G_GNUC_INTERNAL
-void pcmk__new_ordering(pe_resource_t *lh_rsc, char *lh_task,
-                        pe_action_t *lh_action, pe_resource_t *rh_rsc,
-                        char *rh_task, pe_action_t *rh_action,
+void pcmk__new_ordering(pe_resource_t *first_rsc, char *first_task,
+                        pe_action_t *first_action, pe_resource_t *then_rsc,
+                        char *then_task, pe_action_t *then_action,
                         enum pe_ordering type, pe_working_set_t *data_set);
```
Comment (clumens) [inline lib/pacemaker/libpacemaker_private.h:340]: And then there are several in this patch.

**Review by clumens — COMMENTED** [review]
Looks good otherwise.

File: lib/pacemaker/pcmk_sched_location.c (hunk 1 above)
Comment (kgaillot) [reply lib/pacemaker/pcmk_sched_location.c:201]: Yep :)

**Pull request merged**